src/main/java/microsoft/exchange/webservices/data/IAsyncResult.java

```diff
@@ -16,7 +16,7 @@
  * Represents the stauts of Asynchronous operation.
  */
 
-public interface IAsyncResult extends Future {
+public interface IAsyncResult extends Future<Object> {
```

Comment:
perhabs this is useful: [Java Generics - Unbounded Wildcards](http://docs.oracle.com/javase/tutorial/java/generics/unboundedWildcards.html)